### DISCUSSION TITLE
Getting [object object] rendered for row data after updating from alpha-64 to alpha-82

### DISCUSSION THREAD

**Discussion by RedBeardEth** [TanStack/table#3924] [General] [answered]
![image](https://user-images.githubusercontent.com/90423049/168767184-4158ce09-3fb3-44a2-88e1-94e0002c7187.png)

When passing in ReactElements as row data they are now rendering as [object object] - where previously they were rendering the JSX element

**Comment by tannerlinsley** [comment, marked as answer]
Make sure you are using info.getValue() instead of info.value

**Reply by RedBeardEth** [reply]
Thanks, I was missing the cell: in my createDataColumn
